src/Microsoft.Data.Analysis/Microsoft.Data.Analysis.csproj

```diff
@@ -0,0 +1,252 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFramework>netstandard2.0</TargetFramework>
+		<LangVersion>7.3</LangVersion>
+		<AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+		<SuppressFinalPackageVersion>false</SuppressFinalPackageVersion>
+		<VersionPrefix>0.5.0</VersionPrefix>
```

Comment:
We should try to decide how we want to describe package version over in the new repo.

Reply:
You can remove `VersionPrefix` here all together. And Microsoft.Data.Analaysis will just become an "unstable" project in machinelearning. Which means the next time we release it, it will be versioned `0.17.x`.

See

https://github.com/dotnet/machinelearning/blob/8dd47c13bf815d748a4fa78e9313a83827bb2057/eng/BranchInfo.props#L3-L23

Reply:
Hmmm, so 1.5.5 exists in both eng/Versions.props and eng/BranchInfo.props. Anyway, it's not important for our purposes here.

For now, using the version in BranchInfo.props is ok (we'd jump from 0.4.0 -> 0.17.6). But, if we wanted to release a new version of DataFrame, I don't think we'd want to increase the version for all the other projects right?

Reply:
> But, if we wanted to release a new version of DataFrame, I don't think we'd want to increase the version for all the other projects right?

My thoughts were that we would no longer "release a new version of DataFrame" by itself. It would now start shipping along with the rest of the ML.NET libraries on the same cycle. So when `0.17.7` ships next, we would ship `Microsoft.Data.Analysis` v`0.17.7`, same for `0.17.8`, etc.

Reply:
> so 1.5.5 exists in both eng/Versions.props and eng/BranchInfo.props

It looks like arcade will overwrite `VersionPrefix` if `MajorVersion` and `MinorVersion` are set:

https://github.com/dotnet/arcade/blob/ca7fab569267ed3bc73360882d652d119aae5653/src/Microsoft.DotNet.Arcade.Sdk/tools/Version.BeforeCommonTargets.targets#L76

@michaelgsharp - I think we can delete the `VersionPrefix` line in eng/Versions.props.

Reply:
> So when 0.17.7 ships next, we would ship Microsoft.Data.Analysis v0.17.7, same for 0.17.8, etc.

Perfect. I was thinking we should do a quick MDA 0.5.0 release this month and then get on to the ML releases, but it looks like we can release preview packages whenever we want in ML, so this is not a problem.

Reply:
@eerhardt I'll run a test build and see. It would be much better to only have to update in 1 place. My local build is having issues though, so I'll test it when I am able to.